src/utils/maps.ts

```diff
@@ -50,15 +50,16 @@ export function mapNodeToChoice(
 }
 
 export function mapSlugNodeToChoice(
-  nodes: Array<SlugNode & Record<"name", string>>
+  nodes: Array<SlugNode & Record<"name", string> & Record<"boolean", boolean>>
 ): Array<SingleAutocompleteChoiceType | MultiAutocompleteChoiceType> {
   if (!nodes) {
     return [];
   }
 
   return nodes.map(node => ({
     label: node.name,
-    value: node.slug
+    value: node.slug,
+    boolean: node.boolean
```

Comment:
I'm pretty sure it shouldn't be fixed here. Is it used only to map attribute values or somewhere else?